crates/telio-proxy/src/proxy.rs

```diff
@@ -463,7 +463,7 @@ impl Runtime for StateIngress {
 
                 let _ = self.sockets.insert(pk, sock);
             }
-
+            else => pending().await,
```

Comment:
this will never finish - is that what we really want? I would have expected to see maybe an empty branch `else => {}` or maybe in addition `telio_log_warn` in it?

Reply:
I'm debaiting this my self, in general this expected, as we do not lock the order of ingress and egress closing. So we would make QA thing that some error happaned? but warn is not that bad.

empty can cause looping, as closed channel continiously returnes None

Reply:
As this is wait, update would cancel it out

Reply:
I am thinking if we should use option = future => {
match option {}
}

instead of else, and then have more detailed warns.
I would be more allarmed on socket poll failure in comparison to socket_update

Reply:
But will this not invoke the same error (closed branches) as before? I would more for the `else` case with some `telio_log_warn("Dirty exit")` and using `else` generally, this way we can lint out the "else-less" selects.

Reply:
So panic is only invoked if there is some match state not covered, if you capture option, it is fully captured, why it failed now is because we had Some(x) = fut => {...}
But did not have None = fut => {..}, and that caused a panic

Reply:
Yeah, I think that we don't want to lint else-less selects, but non-exhaustive ones. And also pending here looks strange to me, I would just add some log, or just nothing

Reply:
Ok, so will leave other `select!`s as it is.